lib/parseGLTF.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+var fs = require('fs');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+module.exports = parseGLTF;
+
+function parseGLTF(dir, gltfData, callback) {
+    var gltf = JSON.parse(gltfData);
+    gltf.loadedShaders = {};
+
+    var shaders = gltf.shaders;
+    if (defined(shaders)) {
+        for (var shaderId in shaders) {
```

Comment:
For a short function like this where there is only one "id", it is generally better style to use `id` instead of `shaderId`.